architecture_reviews/data_act_pilot/README.md

```diff
@@ -57,7 +54,7 @@ described viewing code as “not real” to be a trap. Their small code base
 quickly grew larger than anticipated and their early technical decisions would
 have long lasting ramifications.
 
-![Diagram matching above description](../../assets/img/data-act-diagram.svg)
+![Diagram matching above description](/architecture_reviews/data_act_pilot/data-act-diagram.png)
 
```

Comment:
If the SVG is being replaced by a PNG, do we still need the SVG or can we delete it?